src/components/LightBox.js

```diff
@@ -28,17 +32,25 @@ class SquareBox extends Component{
         // style={{background: this.state.color[this.state.index] is set the background color base on the index
         // onClick = {this.changeStatus} when the box get clicled, call the changeStatus function
         // {this.state.lightStatus[this.state.index]} show the text status in the box
+        // use Ternary Operator to check the index === 0 or not, I think use true or false in the very beginning might be a little bit easier
         <>
-         <button onClick = {this.addBox} class = "button">
-             Add Light Box
-         </button>
-         <button onClick = {this.removeBox} class = "button">
-             Remove Light Box
-         </button>
-         <div id = "square"  onClick = {this.changeStatus} style={{background: this.state.color[this.state.index]}}>
-             <h2>{this.state.lightStatus[this.state.index]} </h2>
-         </div>
-         
+        
+
+          <div onClick = {this.changeStatus} >
+
+            {this.state.index === 0?
+
+            (<> <img src = {ON} height={50} width={100} alt=""/>
+                <div></div>
+                
+               <img src = {white} height={289} width={230} alt=""/> </>)
+            :
+            (<> <img src = {OFF} height={50} width={100} alt=""/>
+                <div></div>
+                <img src = {yellow} height={289} width={230} alt=""/> </>)
```

Comment:
This is good, however, if you don't want CSS to be involved in your JSX directly you may consider adding a className to each image and create an independent CSS file where you can manipulate style.

Reply:
One last comment, you could add some text inside alt="" to describe the images therefore your app will have better accessibility for the user.

Reply:
I was created a imagesIndex.js in the images folder, somehow it doesn't work. so I have to do this way.

Reply:
and thank you for your time to take a look at code, very good comments. 🤝